sessionctx/variable/session.go

```diff
@@ -1041,6 +1045,25 @@ func (s *SessionVars) setTxnMode(val string) error {
 	return nil
 }
 
+// SetPrevStmtDigest sets the digest of the previous statement.
+func (s *SessionVars) SetPrevStmtDigest(prevStmtDigest string) {
```

Comment:
SessionVars is variable map as a whole, shouldn't implement a specific function for specified var. why not use function like `SetLocal/SystemVar` as unified entrance.

Reply:
No, SessionVars contains context variables of a session, and many functions are specific. Refer to 
`SetStatusFlag/GetStatusFlag`, `SetLastInsertID`, `GetWriteStmtBufs`, `GetReplicaRead` and so on.

Reply:
fine.